cinema_service/settings.py

```diff
@@ -15,7 +15,6 @@
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
```

Comment:
Make sure you've added a blank line at the end of the file as per the code style checklist.